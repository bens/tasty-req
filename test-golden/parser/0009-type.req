1: GET /
---
{ "a": {:t any},
  "b": {:t integer},
  "c": {:t double},
  "d": {:t string},
  "e": {:t boolean},
  "f": {:t object},
  "g": {:t array},
  "h": {:t maybe string},
  }
