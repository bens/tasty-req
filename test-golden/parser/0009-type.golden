[ Command
    { command'id = 1
    , command'always = False
    , command'method = "GET"
    , command'url = [ Right "/" ]
    , command'request_body = Nothing
    , command'response_body =
        Just
          (JsonObject
             (Object
                (fromList
                   [ ( "a" , JsonCustom "t" (Right TyInteger) )
                   , ( "b" , JsonCustom "t" (Right TyDouble) )
                   , ( "c" , JsonCustom "t" (Right TyString) )
                   , ( "d" , JsonCustom "t" (Right TyBool) )
                   , ( "e" , JsonCustom "t" (Right (TyMaybe TyString)) )
                   ])))
    }
]
